--- a/src/bundle_signing.rs	2025-11-23 15:42:04
+++ b/src/bundle_signing.rs	2025-11-23 15:59:04
@@ -607,11 +607,6 @@
             dest_dir.join("Contents")
         };
 
-        self.bundle
-            .identifier()
-            .map_err(AppleCodesignError::DirectoryBundle)?
-            .ok_or_else(|| AppleCodesignError::BundleNoIdentifier(self.bundle.info_plist_path()))?;
-
         let mut resources_digests = settings.all_digests(SettingsScope::Main);
 
         // State in the main executable can influence signing settings of the bundle. So examine
